tests/Datagrid/ListMapperTest.php

```diff
@@ -150,7 +150,7 @@ public function testAddOptionIdentifierWithWrongValue(bool $expected, $value): v
         $this->assertFalse($this->listMapper->has('fooName'));
 
         $this->expectException(\InvalidArgumentException::class);
-        $this->expectExceptionMessageMatches('/^Value for "identifier" option must be boolean, .+ given.$/');
+        $this->expectExceptionMessageRegExp('/^Value for "identifier" option must be boolean, .+ given.$/');
```

Comment:
As in https://github.com/sonata-project/SonataAdminBundle/pull/5939#discussion_r390388279, this was changed to remove warnings for PHPUnit 9

Reply:
@phansys This function was introduced in PhpUnit 8.4 or 8.5. And `make test` actually run withPhpUnit 8.3.

we should finish this PR https://github.com/sonata-project/dev-kit/pull/503 by setting the SYMFONY_PHPUNIT_VERSION too.

Reply:
I've updated sonata-project/dev-kit#503 by setting `SYMFONY_PHPUNIT_VERSION`.

Reply:
> And make test actually run withPhpUnit 8.3.

:thinking: why is that?

```
wget -O phpunit8 https://phar.phpunit.de/phpunit-8.phar
--2020-03-15 09:41:32--  https://phar.phpunit.de/phpunit-8.phar
Resolving phar.phpunit.de (phar.phpunit.de)... 188.94.27.25
Connecting to phar.phpunit.de (phar.phpunit.de)|188.94.27.25|:443... connected.
HTTP request sent, awaiting response... 302 Moved Temporarily
Location: https://phar.phpunit.de/phpunit-8.5.2.phar [following]
--2020-03-15 09:41:32--  https://phar.phpunit.de/phpunit-8.5.2.phar
Reusing existing connection to phar.phpunit.de:443.
HTTP request sent, awaiting response... 200 OK
Length: 2914250 (2.8M) [application/octet-stream]
Saving to: ‘phpunit8’

phpunit8                  100%[=====================================>]   2.78M  1.32MB/s    in 2.1s    

2020-03-15 09:41:34 (1.32 MB/s) - ‘phpunit8’ saved [2914250/2914250]

╭─/tmp 
╰$ chmod +x phpunit8
╭─/tmp 
╰$ ./phpunit8 --version                                   
PHPUnit 8.5.2 by Sebastian Bergmann and contributors.
```

Reply:
And the build seems to be using 8.5.2: https://travis-ci.org/github/sonata-project/SonataAdminBundle/jobs/662539417#L551

Reply:
> > And make test actually run withPhpUnit 8.3.
> 
> 🤔 why is that?
> 
> ```
> wget -O phpunit8 https://phar.phpunit.de/phpunit-8.phar
> --2020-03-15 09:41:32--  https://phar.phpunit.de/phpunit-8.phar
> Resolving phar.phpunit.de (phar.phpunit.de)... 188.94.27.25
> Connecting to phar.phpunit.de (phar.phpunit.de)|188.94.27.25|:443... connected.
> HTTP request sent, awaiting response... 302 Moved Temporarily
> Location: https://phar.phpunit.de/phpunit-8.5.2.phar [following]
> --2020-03-15 09:41:32--  https://phar.phpunit.de/phpunit-8.5.2.phar
> Reusing existing connection to phar.phpunit.de:443.
> HTTP request sent, awaiting response... 200 OK
> Length: 2914250 (2.8M) [application/octet-stream]
> Saving to: ‘phpunit8’
> 
> phpunit8                  100%[=====================================>]   2.78M  1.32MB/s    in 2.1s    
> 
> 2020-03-15 09:41:34 (1.32 MB/s) - ‘phpunit8’ saved [2914250/2914250]
> 
> ╭─/tmp 
> ╰$ chmod +x phpunit8
> ╭─/tmp 
> ╰$ ./phpunit8 --version                                   
> PHPUnit 8.5.2 by Sebastian Bergmann and contributors.
> ```

@VincentLanglet meant when using phpunit bridge, that by default it installs phpunit 8.3

Reply:
> when using phpunit bridge

When using simple-phpunit, right? Cause we are not using it for now AFAIK (in the CI or with `make test`). I think this change can be reverted.

Reply:
> > when using phpunit bridge
> 
> When using simple-phpunit, right? Cause we are not using it for now AFAIK (in the CI or with `make test`). I think this change can be reverted.

It's what I meant 😄to use keep using `expectExceptionMessageMatches`.